### PR TITLE
omalovanky-pdf

### PR DESCRIPTION
**Pull request by KlYana**

### CODE REVIEW

File: pages/kids/pdf/[kidsDictionaryId].tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+interface PhraseImageProps {
+  phrase: Phrase;
+}
+
+const KidsDictionaryPage = ({ dictionary }: InferGetStaticPropsType<typeof getStaticProps>): JSX.Element => {
+  const kidsCategory = useMemo(() => getKidsCategory(dictionary), [dictionary]);
+  const { currentLanguage, otherLanguage } = useLanguage();
+
+  const PhraseImage: FunctionComponent<PhraseImageProps> = ({ phrase }) => {
```
Comment (JuroUhlar) [inline pages/kids/pdf/[kidsDictionaryId].tsx:22]: I would consider moving this outside the Page component and just passing the Kids category as a prop :)

File: pages/kids/pdf/[kidsDictionaryId].tsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        <table className="text-xl font-medium basis-1/2">
+          <tbody>
+            {kidsCategory?.translations
+              .filter((_, index) => index % 2 === 0)
+              .map((phrase) => (
+                <tr key={phrase.getId()}>
+                  <td>
+                    <div className="border-black border-1 h-[840px] mb-2">
+                      <PhraseImage phrase={phrase} />
+                      <div className="p-2 h-[218px] border-t-black border-transparent border-1 flex flex-col items-center justify-center">
```
Comment (JuroUhlar) [inline pages/kids/pdf/[kidsDictionaryId].tsx:44]: Small visual detail: When text fits on one line, it is centered (I guess thanks to flexbox) but when the text wraps to the second line, it is left-aligned (I guess the div is full width then). Try applying `text-align: center` or something to make it always centered :) 
image

**Review by JuroUhlar — CHANGES_REQUESTED** [review]
This is looking awesome, great work! 💪 🚀 
I commented on just two small things

File: pages/kids/pdf/[kidsDictionaryId].tsx (hunk 1 above)
Comment (KlYana) [reply pages/kids/pdf/[kidsDictionaryId].tsx:22]: @JuroUhlar I'm not sure I totally understand. So not to use useMemo and pass kidsCategory as getStaticProps? thanks

File: pages/kids/pdf/[kidsDictionaryId].tsx (hunk 2 above)
Comment (KlYana) [reply pages/kids/pdf/[kidsDictionaryId].tsx:44]: this one should be fixed with new commit

**Review by JuroUhlar — APPROVED** [review]

File: pages/kids/pdf/[kidsDictionaryId].tsx (hunk 1 above)
Comment (JuroUhlar) [reply pages/kids/pdf/[kidsDictionaryId].tsx:22]: Ah sorry for the late reply, I misspoke, I meant extracting the `PhraseImage` function compoenent outside the `KidsDictionaryPage`, I don't see the advantage of defining it in the render method here. But it's not a big deal, I merged the PR, feel free to tweak it in a separate one if you want :)

**Pull request merged**